### PR TITLE
Convert Zesty <Input type='text'> to MUI <TextField>

### LINKED ISSUE #1057: Zesty Input to MUI TextField swap
Skip over checkbox instances and other input types that are different to text

### PR DESCRIPTION
**Pull request by rjpals**
The PR converts all Zesty `<Input>` components (of `type='text'`) into MUI `<TextField>` components.

Closes #1057

### CODE REVIEW

File: src/apps/active-preview/Preview.js [hunk 1]
```diff
@@ -164,10 +164,15 @@ export function Preview(props) {
               >
                 <FontAwesomeIcon icon={faSync} />
               </Button>
-              <Input
+              <TextField
                 ref={input}
-                className={styles.Route}
+                InputProps={{ className: styles.Route }}
```
Comment (agalin920) [inline src/apps/active-preview/Preview.js:169]: What styling is being inherited here? Can it be removed and/or converted to sx?

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js [hunk 2]
```diff
@@ -96,13 +96,20 @@ export const CanonicalTag = memo(function CanonicalTag(props) {
...
                 value={whitelist}
-                className={styles.Input}
                 onChange={handleWhitelist}
                 placeholder="page,category"
+                size="small"
+                variant="outlined"
+                color="primary"
+                sx={{
+                  backgroundColor: "primary.contrastText",
+                  mt: "8px",
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js:110]: Use the system units. mt: 1

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js [hunk 3]
```diff
@@ -114,13 +121,20 @@ export const CanonicalTag = memo(function CanonicalTag(props) {
...
                 value={custom}
                 onChange={handleCustom}
-                className={styles.Input}
                 placeholder="/page/example/ or https://example.com/"
+                size="small"
+                variant="outlined"
+                color="primary"
+                sx={{
+                  backgroundColor: "primary.contrastText",
+                  mt: "8px",
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js:135]: mt: 1

File: src/shell/components/login/Login.js [hunk 4]
```diff
@@ -100,7 +100,19 @@ export default connect((state) => {
...
+                <TextField
+                  name="token"
+                  type="text"
+                  autoComplete="off"
+                  size="small"
+                  variant="outlined"
+                  color="primary"
+                  sx={{
+                    backgroundColor: "primary.contrastText",
+                    mb: "16px",
```
Comment (agalin920) [inline src/shell/components/login/Login.js:112]: mb: 2

**Comment by agalin920** [comment]
I don't think we should go the route of 'use sx prop then eventually remove it once theme gets updated'. Extra work. Remove all the sx props and lets make this ticket dependant on theme udpate.

**Comment by rjpals** [comment]
This PR is now dependent on the merge & release of [the proper style overrides in the theme](https://github.com/zesty-io/material/pull/6)

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js (hunk 2 above)
Comment (shrunyan) [reply src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/CanonicalTag.js:110]: Are `mt` units a material thing? Not seeing them here https://www.w3.org/Style/Examples/007/units.en.html

**Comment by rjpals** [comment]
The necessary theme changes have been merged, published & installed. This PR is now ready to be reviewed and then merged.

**Review by agalin920 — APPROVED** [review]
Approved

**Pull request merged**